packages/dns/dns.0.6.1/opam

```diff
@@ -29,12 +39,11 @@ depends: [
   "ocamlbuild" {build}
   "conf-libssl" {build}
 ]
-depopts: ["mirage-net"]
+depopts: ["mirage-net" "base-unix"]
 conflicts: [
-  "mirage-net" {> "0.9.4"}
-  "mirage-net" {< "0.5.2"}
+  "mirage-net" {< "0.5.2" | > "0.9.4"}
 ]
-dev-repo: "git://github.com/mirage/ocaml-dns"
+available: false # need a constraint on conf-libssl (only in opam 2.1)
```

Comment:
You mean that these were not installable before?

Reply:
sort of -- the underlying OS has upgraded, so it _used_ to be build on an older openssl, but no longer does. When we can express "version of depexts" constraints, this will be buildable again. It's a pity to mark it unavailable when it would techncially build on an older debian, but that cant be helped I think.

Reply:
I don't understand the `(only in opam 2.1)` bit of the comment. What was possible in opam 2.1 that is not in 2.0 that would end up in marking it as available false?